core/math/matrix3.h

```diff
@@ -219,6 +230,8 @@ class Basis {
 	Basis(const Quat &p_quat) { set_quat(p_quat); };
 	Basis(const Vector3 &p_euler) { set_euler(p_euler); }
 	Basis(const Vector3 &p_axis, real_t p_phi) { set_axis_angle(p_axis, p_phi); }
+	Basis(const Vector3 &p_axis, real_t p_phi, const Vector3 &p_scale) { set_axis_angle_scale(p_axis, p_phi, p_scale); }
```

Comment:
Are those new constructors intended to be exposed to GDScript (currently they wouldn't be, see #13151)?

Reply:
If not it's good as is I think.